partcad/src/partcad/ai_feature_file.py

```
@@ -0,0 +1,92 @@
+            type = "image"
+        elif self.is_video:
+            type = "video"
+        elif self.is_pdf:
+            type = "pdf"
+        return f"AiContentFile({self.url}, {type})"
+
+
+class AiContentProcessor:
+    MARKER = "CONTENT_INSERTED_HERE"
+
+    def process_content(self, content: str):
```

Ruff (0.8.2)

Missing return type annotation for public function `process_content`

(ANN201)